firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -559,6 +555,48 @@ final class SearchSettingsTableViewController: ThemedTableViewController, Featur
         cell.selectionStyle = .none
     }
 
+    private func didToggleFirefoxSuggestions(
+        _ toggle: ThemedSwitch,
+        suggestionType: FirefoxSuggestItem
+    ) {
+        var shouldShowSuggestions = false
```

Comment:
```suggestion
        var shouldShowPrivateSuggestionsSetting = false
```
Took me a while to understand what we are doing, but it seems we want to show or hide the private suggestions setting based on the three settings. Maybe it'll help to write a comment or rename?